deploy/helm/sumologic/values.yaml

```diff
@@ -159,6 +159,9 @@ sumologic:
   # watchResourceEventsOverrides:
   #   pods: "v1"
   #   events: "events.k8s.io/v1beta1"
+  
+  ## Option to specify the fluentD buffer as file/memory.
```

Comment:
nit: "fluentD" --> "Fluentd" to match our docs

Reply:
also can we make this
```
fluentd:
  buffer: "memory"
```

Reply:
"FluentD" seems to be used in the `values.yaml`. Should update all those as well?